examples/quantum.py

```diff
@@ -0,0 +1,463 @@
+'''
+This file contains
+* MicroQiskit (https://github.com/qiskit-community/MicroQiskit/blob/master/README.md)
+* `make_line` from QuantumBlur (https://github.com/qiskit-community/QuantumBlur/blob/master/README.md)
+Both are licensed under Apache 2.0, copyright of IBM Research and written by James R. Wootton (@quantumjim)
+Everything else here is under the same license as the rest of the repo.
```

Comment:
I'm going back and forth on this. It's probably ok, but I'm not certain - any thoughts?

Part of me wonders if it would require us to add a caveat in the LICENCE file explaining that it doesn't apply everywhere.

One option could be adding a bit of documentation here about how to put the necessary files somewhere where Python can find them. Another option is keeping the code in your own repository and linking to it in the "Further Work" section of the README we now have - but I realise that one's not ideal.

Thanks for bearing with me. Trying to avoid too many edge cases in the review process so it's clear what we do and don't merge.

Reply:
I think it's probably easier just to tell people to get microqiskit, and remove all the scary words!

Reply:
Really appreciate your understanding! Just to check, the make_line function is ok under MIT now? If so, I think this is good to merge 🎉

Reply:
Yep, that's fine to be under MIT.